### ISSUE TITLE
Navigation heading levels should only increase by one

### ISSUE THREAD

**Issue by alexgibson** [mozilla/bedrock#15809]
# Axe-core® Accessibility Results for component (navigation-desktop) project

Page URL: [https://dev.bedrock.nonprod.webservices.mozgcp.net/en-US/?automation=true](https://dev.bedrock.nonprod.webservices.mozgcp.net/en-US/?automation=true)  

## axe-core found 1 violation

| #   | Description | Axe rule ID | WCAG | Impact | Count |
| --- | --- | --- | --- | --- | --- |
| [1](#1) | Heading levels should only increase by one | heading-order | Best practice | moderate | 1   |

### Failed

1. Heading levels should only increase by one

[Learn more](https://dequeuniversity.com/rules/axe/4.10/heading-order?application=playwright)

heading-order

Best practice

Ensure the order of headings is semantically correct

moderate

Issue Tags: cat.semantics best-practice

| #   | Issue Description | To solve this violation, you need to... |
| --- | --- | --- |
| 1   | **Element location**<br><br>```<br>a[href$="new/"] > h4<br>```<br><br>**Element source**<br><br>```<br><h4 class="m24-c-menu-item-title">Firefox for Desktop</h4><br>``` | Fix any of the following:<br><br>*   Heading order invalid |
